appcli/orchestrators.py

```diff
@@ -577,6 +586,74 @@ def __exec_command(self, command: Iterable[str]) -> CompletedProcess:
         return subprocess.run(command, capture_output=True)
 
 
+class NullOrchestrator(Orchestrator):
+    """
+    Orchestrator for standalone applications.
+    """
+
+    def start(
+        self, cli_context: CliContext, service_name: tuple[str, ...] = None
+    ) -> CompletedProcess:
+        logger.info("NullOrchestrator has no services to start.")
+        return None
+
+    def shutdown(
+        self, cli_context: CliContext, service_name: tuple[str, ...] = None
+    ) -> CompletedProcess:
+        logger.info("NullOrchestrator has no services to shutdown.")
+        return None
+
+    def status(
+        self, cli_context: CliContext, service_name: tuple[str, ...] = None
+    ) -> CompletedProcess:
+        logger.info("NullOrchestrator has no services to get the status of.")
+        return None
+
+    def task(
+        self,
+        cli_context: CliContext,
+        service_name: str,
+        extra_args: Iterable[str],
+        detached: bool = False,
+    ) -> CompletedProcess:
+        logger.info("NullOrchestrator has no services to run tasks for.")
+        return None
+
+    def exec(
+        self,
+        cli_context: CliContext,
+        service_name: str,
+        command: Iterable[str],
+        stdin_input: str = None,
+    ) -> CompletedProcess:
+        logger.info(
+            "NullOrchestrator has no running containers to execute commands in."
+        )
+        return None
+
+    def get_logs_command(self) -> click.Command:
+        @click.command()
+        def log():
+            logger.info("NullOrchestrator has no services to get logs of.")
+            return None
+
+        return log
+
+    def get_name(self) -> str:
+        return "null"
+
+    def verify_service_names(
+        self, cli_context: CliContext, service_names: tuple[str, ...]
+    ) -> bool:
+        if service_names is None or len(service_names) == 0:
+            return True
+        logger.info("NullOrchestrator has no services.")
+        return False
+
+    def get_disabled_commands(self) -> list[str]:
+        return ["init", "service", "task"]
```

Comment:
Add a comment explaining why each of these commands makes no sense, i.e. why we are disabling them.